namespace-provisioner/how-tos.hbs.md

```diff
@@ -177,7 +177,7 @@ namespace_provisioner:
     path: _ytt_lib/workload-sa
 ```
 
-Sample customization (`.lib.yaml`) file for overriding regoFile of the default ScanPolicy
+Sample customization (`.lib.yaml`) file for overriding the `secrets` and `imagePullSecrets` of the default ServiceAccount
 - [Link to the Sample file](https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/namespace-provisioner-gitops-examples/default-resources-overrides/overlays/scanpolicy-rego.lib.yaml)
```

Comment:
Link is wrong. Should be https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/namespace-provisioner-gitops-examples/default-resources-overrides/overlays/sa-secrets.lib.yaml

Reply:
Changed